genome_designer/main/static/js/contig_view.js

```diff
@@ -30,6 +32,21 @@ gd.ContigView = Backbone.View.extend({
     formJqueryObj.submit();
   },
 
+   /** Makes jbrowse tracks for contig and redirects to jbrowse. */
+  handleJbrowse: function() {
+
+    var getData = {
+        contigUid: INIT_JS_DATA.entity.uid
+    };
+
+    this.enterLoadingState();
+
+    $.get('/_/contigs/make_contig_jbrowse_tracks', getData,
+          _.bind(function(response) {
+              this.exitLoadingState();
+              window.location.href = JBROWSE_LINK;}, this))
```

Comment:
minor style nit:

    _.bind(function(response) {
        this.exitLoadingState();
        window.location.href = JBROWSE_LINK;
    }, this));

**remember the missing semi-colon**